packages/cli/src/controllers/e2e.controller.ts

```diff
@@ -112,14 +113,8 @@ export class E2EController {
 	}
 
 	@Post('/push')
-	async push(req: PushRequest) {
-		const pushInstance = Container.get(Push);
-
-		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-		// @ts-ignore
-		const sessionId = Object.keys(pushInstance.getBackend().connections as object)[0];
-
-		pushInstance.send(req.body.type, req.body.data, sessionId);
+	async pushSend(req: PushRequest) {
+		this.push.send(req.body.type, req.body.data, req.body.sessionId);
```

Comment:
looks like this change will fail because despite adding `sessionId` to the backend type, the e2e code doesn't actually seem to be sending the `sessionId`.
But, frontend also sends `sessionId` as a header on every rest request, which we don't use in the backend 🤷🏽